eZ/Publish/Core/settings/policies.yml

```diff
@@ -39,7 +39,7 @@ parameters:
         section:
             assign: { Class: true, Section: true, Owner: true, NewSection: true }
             edit: ~
-            view: ~
+            view: { Subtree: false, Section: false }
```

Comment:
It works, and does not negatively affect other policies, but:
1. this is config for policy limitations, not role limitations
2. the issue QA reported here is probably present in several parts of the system, basically anything using hasAccess() and not expecting limitations back even tough the system is extendable so someone might add it